flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -717,21 +717,21 @@ class TableEnvironmentTest {
     assertThat(ResolvedSchema.of(Column.physical("database name", DataTypes.STRING())))
       .isEqualTo(tableResult5.getResolvedSchema)
     assertThat(CollectionUtil.iteratorToList(tableResult5.collect).toArray())
-      .isEqualTo(util.Arrays.asList(Row.of("db1"), Row.of("db2")).toArray())
+      .containsExactly(util.Arrays.asList(Row.of("db1"), Row.of("db2")).toArray())
 
     val tableResult6 = tableEnv.executeSql("SHOW DATABASES LIKE '_re%'")
     assertThat(tableResult6.getResultKind).isEqualTo(ResultKind.SUCCESS_WITH_CONTENT)
     assertThat(ResolvedSchema.of(Column.physical("database name", DataTypes.STRING())))
       .isEqualTo(tableResult6.getResolvedSchema)
     assertThat(CollectionUtil.iteratorToList(tableResult6.collect).toArray())
-      .isEqualTo(util.Arrays.asList(Row.of("pre_db3")).toArray())
+      .containsExactly(util.Arrays.asList(Row.of("pre_db3")).toArray())
 
     val tableResult7 = tableEnv.executeSql("SHOW DATABASES NOT LIKE '_re%'")
     assertThat(tableResult7.getResultKind).isEqualTo(ResultKind.SUCCESS_WITH_CONTENT)
     assertThat(ResolvedSchema.of(Column.physical("database name", DataTypes.STRING())))
       .isEqualTo(tableResult7.getResolvedSchema)
     assertThat(CollectionUtil.iteratorToList(tableResult7.collect).toArray())
-      .isEqualTo(util.Arrays.asList(
+      .containsExactly(util.Arrays.asList(
```

Comment:
no need for `Arrays.asList`, you can use a vararg here

Reply:
OK